tests/unit/btree_test_common.c

```diff
@@ -49,6 +49,20 @@ init_clockcache_config_from_master_config(clockcache_config *cache_cfg,
    return 1;
 }
 
+int
+init_task_config_from_master_config(task_system_config  *task_cfg,
+                                    const master_config *master_cfg,
+                                    uint64               scratch_size)
+{
+   platform_status rc;
+   uint64          num_bg_threads[NUM_TASK_TYPES] = {0};
+   num_bg_threads[TASK_TYPE_NORMAL]   = master_cfg->num_normal_bg_threads;
+   num_bg_threads[TASK_TYPE_MEMTABLE] = master_cfg->num_memtable_bg_threads;
+   rc                                 = task_system_config_init(
```

Comment:
Minor request: If you can insert a blank line before this one, this fn-call will look like a function call.